nav2_costmap_2d/plugins/inflation_layer.cpp

```diff
@@ -90,6 +92,42 @@ InflationLayer::onInitialize()
   need_reinflation_ = false;
   cell_inflation_radius_ = cellDistance(inflation_radius_);
   matchSize();
+  setParamCallbacks();
+}
+
+void
+InflationLayer::setParamCallbacks()
```

Comment:
Does this work through rqt reconfigure now?

Reply:
I haven't tested that yet